test/integration/host_js_test.rb

```diff
@@ -302,6 +302,26 @@ class HostJSTest < IntegrationTestWithJavascript
       assert_current_path hosts_path
       assert_empty(page.driver.cookies['_ForemanSelectedhosts'])
     end
+
+    test 'select all hosts in all pages' do
+      Setting[:entries_per_page] = 1
+      # We need to create a second host for 2 pages to show up
+      FactoryGirl.create(:host)
+      visit hosts_path
+      page.find('#check_all').trigger('click')
+      # Ensure and wait for all hosts to be checked, and that no unchecked hosts remain
+      assert page.has_no_selector?('input.host_select_boxes:not(:checked)')
+      # Suggestion dialog shows up once we click on 'check all'
+      assert page.has_selector? "#multiple-alert"
+      select_all_hosts = "Select all #{Host.unscoped.count} hosts"
+      assert page.has_link? select_all_hosts
+      # Multiple is false when we have not clicked "select all"
```

Comment:
True or false? The assertion below checks whether it's true, and later it seems to be false.